sdk/eventhub/azure-eventhub/tests/livetest/asynctests/test_auth_async.py

```diff
@@ -45,6 +45,7 @@ def on_event(partition_context, event):
     assert on_event.called is True
     assert on_event.partition_id == "0"
     assert list(on_event.event.body)[0] == 'A single message'.encode('utf-8')
+    await credential.close()
```

Comment:
I didn't have to do this before?

Reply:
it's still not needed, I add it to avoid warning.

when I was testing this case, I was warning logs like "transport pipeline session not closed" which indicates the aio credential is not closed. so I add it

Reply:
Gotcha. So a customer would not normally need to call `close` on their credentials - this would be in the client?

Reply:
the event hub client won't call `credential.close` internally.
the life cycle of the `credential` is managed by users themselves.